### PR TITLE
[PowerMock] LPS 151823

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi  @dantewang @tinatian,

This pr is to remove powermock usages present in `BaseDDMFormFieldTypeSettingsTestCase` and its related class located in `dynamic-data-mapping-form-field-type`. Also see https://github.com/JanisZhang/liferay-portal/pull/21

https://issues.liferay.com/browse/LPS-151823 

Thanks for checking.

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributorTest.java [hunk 1]
```diff
@@ -209,10 +240,33 @@ public void testGetParametersForSignedInUser() {
 			documentLibraryDDMFormFieldTemplateContextContributor = _createSpy(
 				themeDisplay);
 
+		DDMFormFieldRenderingContext ddmFormFieldRenderingContext =
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributorTest.java:243]: @JanisZhang Why do you need to add this logic when there's already `_setUpItemSelector()` in `setUp()`?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributorTest.java [hunk 2]
```diff
@@ -375,10 +433,35 @@ public void testGetParametersWithNullGroupShouldContainItemSelectorURL() {
 			documentLibraryDDMFormFieldTemplateContextContributor = _createSpy(
 				themeDisplay);
 
+		// janis
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributorTest.java:436]: @JanisZhang Typo?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributorTest.java (hunk 1 above)
Comment (JanisZhang) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributorTest.java:243]: @dantewang sorry, this logic is incorrect, I don't want to add this commit in the pull... forgot to drop it.

**Comment by JanisZhang** [comment]
ci:close

**Pull request closed without merging**